trace-dispatcher/src/Cardano/Logging/Tracer/EKG.hs

```diff
@@ -0,0 +1,94 @@
+{-# LANGUAGE RankNTypes          #-}
+{-# LANGUAGE RecordWildCards     #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Cardano.Logging.Tracer.EKG (
+  ekgTracer
+) where
+
+import           Cardano.Logging.DocuGenerator
+import           Cardano.Logging.Types
+
+import           Control.Monad.IO.Class (MonadIO, liftIO)
+import qualified Control.Tracer as T
+import           Data.IORef (newIORef, readIORef, writeIORef, IORef)
+import qualified Data.Map.Strict as Map
+import           Data.Text (pack, Text)
+import qualified System.Metrics as Metrics
+import qualified System.Metrics.Counter as Counter
+import qualified System.Metrics.Gauge as Gauge
+import qualified System.Metrics.Label as Label
+import           System.Remote.Monitoring (Server, getCounter, getGauge,
+                     getLabel)
+
+
+ekgTracer :: MonadIO m => Either Metrics.Store Server-> m (Trace m FormattedMessage)
```

Comment:
@karknu I think you'd like to check this ekg tracer.  I think you'd prefer to have a tracer which does not need to make a namespace lookup but has an access to the gauge directly, which would make it very fast.

Reply:
Just did a benchmark with this solution, and it sends 1000 messages, each containing 5 metrics in less then 2 microseconds in the current non optimized version:)

Reply:
@jutaro, how does it compare to direct submission?

Reply:
That sounds like it fast enough.

Reply:
Was the map was almost empty? But even with `100`'s entries it shouldn't be that much slower.